docs/modules/clients/pages/clc.adoc

```diff
@@ -0,0 +1,350 @@
+= Hazelcast Command-Line Client
+:page-api-reference: https://github.com/hazelcast/hazelcast-commandline-client
+:page-toclevels: 3
+[[clc-client]]
+
+Hazelcast Command-Line Client or *hzc* in short, is a command-line tool.
```

Comment:
I would change it to: "Hazelcast Command Line Client or CLC ..."

Reply:
Are we calling it hzc for short? Shouldn't we call it Hazelcast CLC? Since this tool also includes commands for Hazelcast Cloud, which uses hzcloud.

I'd recommend something like this for the overview:

> The Hazelcast Command-Line Client (CLC) is a cross-platform command-line tool to connect to Hazelcast Platform and Hazelcast Cloud. You can use the CLI to execute commands through a terminal, using interactive command-line prompts, or a script.

> You can install the Hazelcast CLC locally on Linux or Mac computers.

Reply:
@JakeSCahill your recommendation looks great. I think we should drop cross-platform for now since Windows is not supported yet.

I think it also misses the actual description of the tool. We want users to mainly use it to operate on distributed data structures. There is also just few cluster utility commands. I think we need to describe this too somehow. Wdyt?

Reply:
Refactored Jake's suggestion a bit. Let me know if it doesn't work, please. https://github.com/hazelcast/hz-docs/pull/462/commits/b3e94f20a0864ac722d6b633d8dc5a57e9758e7b